stack/dashboards/rpm/wazuh-dashboards.spec

```diff
@@ -0,0 +1,420 @@
+fi
+
+# Stop the services to upgrade the package
+if [ $1 = 2 ]; then
+  if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1 && systemctl is-active --quiet wazuh-dashboards > /dev/null 2>&1; then
+    systemctl stop wazuh-dashboards.service > /dev/null 2>&1
+    touch %{INSTALL_DIR}/wazuh-dashboards.restart
+  # Check for SysV
+  elif command -v service > /dev/null 2>&1 && service wazuh-dashboards status 2>/dev/null | grep "is running" > /dev/null 2>&1; then
+    service wazuh-dashboards stop > /dev/null 2>&1
+    touch %{INSTALL_DIR}/wazuh-dashboards.restart
+  fi
+fi
+
+# -----------------------------------------------------------------------------
+
+%post
+setcap 'cap_net_bind_service=+ep' %{INSTALL_DIR}/node/bin/node
+
+# -----------------------------------------------------------------------------
+
+%preun
+if [ $1 = 0 ];then # Remove
+  echo -n "Stopping wazuh-dashboards service..."
+  if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1 && systemctl is-active --quiet wazuh-dashboards > /dev/null 2>&1; then
+      systemctl stop wazuh-dashboards.service > /dev/null 2>&1
+  # Check for SysV
+  elif command -v service > /dev/null 2>&1 && service wazuh-dashboards status 2>/dev/null | grep "is running" > /dev/null 2>&1; then
+    service wazuh-dashboards stop > /dev/null 2>&1
+  fi
+fi
+
+# -----------------------------------------------------------------------------
+
+%postun
+if [ $1 = 0 ];then
+  # If the package is been uninstalled
+  if [ $1 = 0 ];then
+    # Remove the wazuh-dashboards user if it exists
+    if id -u %{USER} > /dev/null 2>&1; then
+      userdel %{USER} >/dev/null 2>&1
+    fi
+    # Remove the wazuh-dashboards group if it exists
+    if command -v getent > /dev/null 2>&1 && getent group %{GROUP} > /dev/null 2>&1; then
+      groupdel %{GROUP} >/dev/null 2>&1
+    elif id -g %{GROUP} > /dev/null 2>&1; then
+      groupdel %{GROUP} >/dev/null 2>&1
+    fi
+  fi
+
+  # Remove /etc/wazuh-dashboards and /usr/share/wazuh-dashboards dirs
+  rm -rf %{CONFIG_DIR}
+  rm -rf %{INSTALL_DIR}
+  rm -rf %{PID_DIR}
+  if [ -d %{LOG_DIR} ]; then
+      rmdir --ignore-fail-on-non-empty %{LOG_DIR}
+  fi
+fi
+
+# -----------------------------------------------------------------------------
+
+# posttrans code is the last thing executed in a install/upgrade
+%posttrans
+if [ ! -d %{LOG_DIR} ]; then
+    mkdir -p %{LOG_DIR}
+    chown %{USER}:%{GROUP} %{LOG_DIR}
+fi
+if [ ! -d %{PID_DIR} ]; then
+    mkdir -p %{PID_DIR}
+    chown %{USER}:%{GROUP} %{PID_DIR}
+fi
+
+if [ -f %{INSTALL_DIR}/wazuh-dashboards.restart ]; then
+  rm -f %{INSTALL_DIR}/wazuh-dashboards.restart
+  if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1 ; then
+    systemctl daemon-reload > /dev/null 2>&1
+    systemctl restart wazuh-dashboards.service > /dev/null 2>&1
+  elif command -v service > /dev/null 2>&1 && service wazuh-dashboards status 2>/dev/null | grep "running" > /dev/null 2>&1; then
```

Comment:
dead code, the dashboards never will be running on this step, you stopped it in preun

Reply:
Done:[02a29f](https://github.com/wazuh/wazuh-packages/commit/02a29f8a7bdf3a69b9e485ca2f4d787337a8c06f)